core/src/main/java/com/linecorp/armeria/client/ConnectionPoolListener.java

```diff
@@ -71,9 +75,13 @@ static ConnectionPoolListener logging(Ticker ticker) {
      *   <td>The number of closed connections.</td>
      * </tr>
      * <tr>
-     *   <td>{@code armeria.client.active.connections#value}</td>
+     *   <td>{@code armeria.client.connections.pool#value{state="active"}}</td>
```

Comment:
What do you think about this name?

- `armeria.client.connection.pool.size`

Reply:
That would be clearer!
```
armeria.client.connection.pool.size#value{state="active"}
armeria.client.connection.pool.size#value{state="idle"}
```

But looks like there is a similar metric naming(armeria.client.connections). Should I have to change this too?
```
armeria.client.connections.lifespan
```